t/09_qesapdeployment.t

```diff
@@ -966,4 +966,30 @@ subtest '[qesap_is_job_finished]' => sub {
     ok($results[2] == 0, "Consider 'running' if the openqa job status response is 'running'");
 };
 
+subtest '[qesap_az_get_native_fencing_type] empty or undefined openqa variable' => sub {
```

Comment:
usually we are doing it with pattern like
```
set_var('SOMETHING', 'somevalue');
my $res_undef = qesap_something_to_test();
set_var('SOMETHING', undef);
```

So in this case the test should looks like
```
subtest '[qesap_az_get_native_fencing_type] empty or undefined openqa variable' => sub {
    my $res = qesap_az_get_native_fencing_type();
    ok($res eq 'msi', "Return 'msi' if openqa var is empty or undef");
}
```

Also distinguishing between `undef` and `empty` is not something that qesapdeployment.pm is doing but something provided internally by the `get_var` itself, so out of scope in test here.